pkg/_pub_shared/lib/data/package_api.dart

```diff
@@ -90,18 +90,28 @@ class GithubPublishingConfig {
   /// The GitHub environment where the publishing is required from.
   String? environment;
 
+  /// Whether the (default) `push` event is disabled.
+  bool? disablePushEvent;
+
+  /// Whether the `workflow_dispatch` event is enabled.
+  bool? enableWorkflowDispatchEvent;
```

Comment:
Let's give all the boolean value default values

Reply:
PTAL.